### ISSUE TITLE
[Bug] Console warning on dbc.Tooltip component

### ISSUE THREAD

**Issue by nadijagraca** [facultyai/dash-bootstrap-components#1004]
Environment:
- dash version: ` 2.15.0`
- dash-bootstrap-components version: `1.4.2`
- components affected by bug: dbc.Tooltip

### What is happening?

When I use dbc.Tooltip component I get the below warning in the console. 

<img width="408" alt="Screenshot 2024-03-14 at 16 30 01" src="https://github.com/facultyai/dash-bootstrap-components/assets/108531476/394a074f-0d2a-49c0-9cd7-c8aee9bc6c96">

### What should be happening?

No warning. 

### Code

```python
from dash import Dash, html
import dash_bootstrap_components as dbc

app = Dash(__name__)

app.layout = html.Div([dbc.Button("Button", id="button"), dbc.Tooltip("Tooltip text", target="button")])

if __name__ == '__main__':
    app.run(debug=True)
```

### Error messages

```bash
react-dom@16.v2_15_0…09203071.14.0.js:82 Warning: React does not recognize the `hasDoneInitialMeasure` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `hasdoneinitialmeasure` instead. If you accidentally passed it from a parent component, remove it from the DOM element
```

**Comment by tcbegley** [comment]
Thanks for reporting this, and sorry for slow response. I was able to reproduce this, but so far I've not managed to track down where the error is actually coming from. The `hasDoneInitialMeasure` prop is not being passed around by any of our code, so perhaps we're using a dependency incorrectly or something. 

I'll keep trying to figure it out in the meantime.

**Comment by tcbegley** [comment]
Ok, almost as soon as I posted about this in defeat I figured it out 😅 

Fix implemented in #1008.

**Comment by tcbegley** [comment]
Fixed in the latest version

```
pip install -U dash-bootstrap-components
```
